src/index.html

```diff
@@ -7,8 +7,40 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a class="img" href="index.html">
+        <img src="./images/logo.png" alt="Logo">
+      </a>
+
+      <nav class="container">
+        <ul class="nav__list">
+          <li><a class="is-active link" href="index.html">Apple</a></li>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) 5th item. Don't put 2 tags in one line, this way code is really hard to read. It should look something like that
```suggestion
          <li>
            <a class="is-active link" href="index.html">Apple</a>
          </li>
```
Fix it everywhere